[docs][Slider] Add range slider example (#1600)
mui/base-ui: docs/src/app/(public)/(content)/react/components/slider/page.mdx

```

<Reference component="Slider" parts="Root, Value, Control, Track, Indicator, Thumb" />

## Examples

### Range slider

> To let users set the start and end of a range on a Slider, provide an array of values to the `value` or `defaultValue` prop on root.

Reviewer: ```suggestion
To create a range slider, provide an array of values with corresponding thumbs:
```
